### PR TITLE
Update_base_templates

### PR DESCRIPTION
**Pull request by tonybaloney**
## Purpose
 Sync the core folder with the azd core templates (with 1 minor change for postgres service ID in ACA

### CODE REVIEW

**Comment by tonybaloney** [comment]
There are some breaking changes in here. Do not merge

**Comment by tonybaloney** [comment]
See related docs https://learn.microsoft.com/en-us/azure/templates/microsoft.app/containerapps?pivots=deployment-language-bicep#secret it needs to support name:value, or name: `identity`, `keyVaultUrl`

File: {{cookiecutter.__src_folder_name}}/infra/core/host/container-app.bicep [hunk 1]
```diff
@@ -130,7 +130,9 @@ resource app 'Microsoft.App/containerApps@2023-05-02-preview' = {
       } : { enabled: false }
       secrets: [for secret in items(secrets): {
```
Comment (tonybaloney) [inline {{cookiecutter.__src_folder_name}}/infra/core/host/container-app.bicep:131]: This is different to the base templates, I can't see another way unless there's a method of not doing keyVaultUrl

Comment (pamelafox) [reply {{cookiecutter.__src_folder_name}}/infra/core/host/container-app.bicep:131]: Looks good as long as its okay with the null values!

**Review by pamelafox — APPROVED** [review]

**Pull request merged**